CONTRIBUTING.md

```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for [Styleguide Website](https://playbook.cio.gov/designstandards)
+
+If you'd like to see the changes to the Draft Web Design Standards in the local
+version of the website, you will need to run the `local-gem` gulp task before
+the `website:serve` task. This flag will generate a local version of the Ruby
+gem that the Jekyll site will use instead of the version described in the
+`Gemfile`.
+
+```sh
+gulp clean-dist && \ # Cleans up `dist-gem` if it exists
+gulp local-gem website:serve
```

Comment:
I'm using these instructions and I'm not seeing my updated code on the local website. Is there something else that might have to be done here?